crates/runtime/src/accelerated_table.rs

```diff
@@ -73,22 +110,69 @@ impl AcceleratedTable {
             accelerator,
             federated,
             refresh_handle,
+            refresh_mode,
+            refresh_trigger,
+            _scheduled_refreshes_handle: scheduled_refreshes_handle,
```

Comment:
This should be added to the `impl Drop` section to be aborted when the accelerated table is dropped.

Reply:
Thank you, updated